en/releases/main.md

```diff
@@ -54,3 +54,9 @@ TBD ...
 ### Fixed-wing
 
 - TBD
+
+### ROS 2
+
+- [Experimental] [PX4 ROS 2 Interface Library](../ros2/px4_ros2_interface_lib.md): A new C++ library that simplifies controlling PX4 from ROS 2.
+  Supports adding flight modes in ROS 2 that are peers of the PX4 modes running on the flight controller.
+  Added to PX4 in [PX4-Autopilot#20707](https://github.com/PX4/PX4-Autopilot/pull/20707) (initial support).
```

Comment:
FYI the important thing here is to set expectation. So I have put a prefix experimental and added a link to the initial PR. 
As you enhance this you can add new PR links. 
If you consider significant parts of this stable, we can remove "Experimental".